From 99baaf2121ca40a71a65499e16af666621e9f005 Mon Sep 17 00:00:00 2001
From: Stefan Roese <sr@denx.de>
Date: Fri, 4 Jan 2019 15:36:34 +0100
Subject: [PATCH] WIP: MIPS: ralink: dts: gardena_smart_gateway_mt7688: Set
 netdev as default for eth LED's

This patch set's "netdev" as default for both ethernet LED's (link and
activity) via device-tree.

Signed-off-by: Stefan Roese <sr@denx.de>
---
 arch/mips/boot/dts/ralink/gardena_smart_gateway_mt7688.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/mips/boot/dts/ralink/gardena_smart_gateway_mt7688.dts b/arch/mips/boot/dts/ralink/gardena_smart_gateway_mt7688.dts
index 3ce7a549e8c4..c9ae52b3ef56 100644
--- a/arch/mips/boot/dts/ralink/gardena_smart_gateway_mt7688.dts
+++ b/arch/mips/boot/dts/ralink/gardena_smart_gateway_mt7688.dts
@@ -74,13 +74,13 @@
 		ethernet_link {
 			label = "smartgw:eth:link";
 			gpios = <&gpio0 3 GPIO_ACTIVE_LOW>;
-			default-state = "off";
+			linux,default-trigger = "netdev";
 		};
 
 		ethernet_activity {
 			label = "smartgw:eth:act";
 			gpios = <&gpio1 11 GPIO_ACTIVE_LOW>;
-			default-state = "off";
+			linux,default-trigger = "netdev";
 		};
 	};
 
